dev/TeachingTip/TeachingTip.cpp

```diff
@@ -1375,20 +1383,22 @@ winrt::TeachingTipPlacementMode TeachingTip::DetermineEffectivePlacement()
 
 void TeachingTip::EstablishShadows()
 {
-#ifdef USE_INTERNAL_SDK
+#ifdef USE_INSIDER_SDK
     if (SharedHelpers::IsThemeShadowAvailable())
```

Comment:
Please use the IUIElement10 check that is present elsewhere in the code, we should probably delete IsThemeShadowAvailable helper. Something like:
```suggestion
    if (winrt::IUIElement10 beakPolygon_uiElement10 = m_beakPolygon.get())
```

and then later use `beakPolygon_uiElement.Shadow(...)`. This is important because otherwise when running on older 19H1 insider builds the vtable of IUIElement10 won't line up and we crash.